lib/core/text/abstract_text.nit

```diff
@@ -1489,16 +1485,32 @@ abstract class Buffer
 	#     b = new FlatBuffer.from("ab_c -ab0c ab\nc")
 	#     b.capitalize
 	#     assert b == "Ab_C -Ab0C Ab\nC"
-	fun capitalize do
+	#
+	#     b = new FlatBuffer.from("12345")
+	#     b.capitalize(source="foo")
+	#     assert b == "Foo45"
+	#
+	#     b = new FlatBuffer.from("preserve my ACRONYMS")
+	#     b.capitalize(keep_upper=true)
+	#     assert b == "Preserve My ACRONYMS"
+	fun capitalize(source: nullable Text, keep_upper: nullable Bool) do
```

Comment:
I first thought to make it private, like `internal_capitalize(source: Text, keep_upper: Bool)` but #2062 made me think that it could be useful to reuse buffers.